src/components/common/SessionApprove.vue

```diff
@@ -75,15 +75,16 @@ import { required } from "vuelidate/lib/validators"
 // TODO move into own helper file
 // Parse into Lunie tx format from signMessage to display tx properly
 function parseTx(signMessage) {
-  const { msgs, fee, memo } = JSON.parse(signMessage)
+  const { msgs, fee, memo, delegates } = JSON.parse(signMessage)
 
   return {
     tx: {
       type: "auth/StdTx",
       value: {
         msg: msgs,
         fee,
-        memo
+        memo,
+        delegates
```

Comment:
this doesn't look correct. the delegates which serve as meta data shouldn't be part of the actual transaction